.circleci/config.yml

```diff
@@ -99,6 +99,9 @@ jobs:
     executor:
       name: amphtml-docker-small-executor
     steps:
+      - run:
+          name: 'Create Workspace'
+          command: mkdir -p /tmp/workspace
```

Comment:
I'd imagine this will also need to be removed from `compute_merge_commit.sh`?

Reply:
Dangling